asteroid/complex_nn.py

```diff
@@ -54,23 +58,27 @@ def as_torch_complex(x, asteroid_dim: int = -2):
         ValueError: If type of `x` is not understood.
     """
     if isinstance(x, (list, tuple)) and len(x) == 2:
-        return torch_complex_from_magphase(*x)
+        return torch_complex_from_reim(*x)
```

Comment:
I'd be in favour of removing it completely as it will be confusing if it works from mag and phase or from real and imag. 
And using either of `torch_complex_from*` will be more explicit.

Reply:
Or rather, rename it to `torch_complex_from_reim`?